src/appengine/handlers/cron/triage.py

```diff
@@ -250,11 +257,105 @@ def _check_and_update_similar_bug(testcase, issue_tracker):
   return False
 
 
-def _file_issue(testcase, issue_tracker):
+def _query_job_bugs_filed_count(job_type, valid_timestamp):
```

Comment:
We need to do these at most once per job and project. Otherwise this will be too slow.